src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -1976,6 +2058,29 @@ module Types = struct
                 Error "Invalid format for token."
           with _ -> Error "Invalid format for token.")
 
+    (* TODO: define a type otherwise identical to Party.Signed.t, but
+       which makes the nonce optional
+    *)
+    let snapp_party_signed_arg : Party.Signed.t option arg_typ =
+      scalar "SnappPartySigned"
+        ~doc:"A party to a Snapp transaction with a signature authorization"
+        ~coerce:(fun party_signed ->
+          let json = to_yojson party_signed in
+          Party.Signed.of_yojson json)
+
```

Comment:
This is fine for now, but we'll want to manually expand the json as a fully-formed GraphQL query before release, so that it's exposed in the schema, docs, etc. Can you open an issue for this?

Reply:
I think we should do it in this PR if we want it before release

Reply:
@mrmr1993 Do you mean, expose the field names in GraphQL? Same for the other pieces of the `Parties.t`?

Reply:
Yea